main.py

```diff
@@ -13,25 +13,21 @@
 os.environ["TF_XLA_FLAGS"] = "--tf_xla_enable_xla_devices"
```

Comment:
Adding all comments in this file since Github won't let me access unchanged files for review.
model.py - adam optimizer - I noticed you are using the default adam optimizer.  It can be customized which may help your algorithm.  It is unlikely it is perfect straight out of the box for your application.  https://keras.io/api/optimizers/adam/

Reply:
Hi Brittney. Thanks! we didn't know that. We're not exactly sure what all the parameters do, so while we're looking over the Adam optimizer, Connor is going to train a new model with double the learning rate parameter just to see what happens. Is there anything else you noticed that we should fix/improve?

Thanks, 
Kevin & Connor